### PR TITLE
Make the readme file more valuable

### LINKED ISSUE #3: Our second issue

### LINKED ISSUE #4: Our third issue

### PR DESCRIPTION
**Pull request by raunaq-r**
Created a much better readme layout as well as an example readme layout so that we can better inform people about what our repository is all about.
Fixes #3

### CODE REVIEW

File: README.md
Comment (raunaq-dev) [inline README.md:1]: Looks good to me

**Review by raunaq-dev — CHANGES_REQUESTED** [review]
Address what I said

**Review by raunaq-r — COMMENTED** [review]
I think I'm done

**Review by raunaq-dev — APPROVED** [review]
Ok, fine it look goods now

**Comment by raunaq-ranjan** [comment]
Fixes #3

**Pull request merged**